rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -635,3 +635,76 @@ def test_query_new_events(rotkehlchen_api_server_with_exchanges: 'APIServer') ->
             query_filter=query_filter,
         )
         assert kraken_events_count != 0
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [0])
+def test_add_edit_asset_movements(rotkehlchen_api_server: 'APIServer') -> None:
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    db = DBHistoryEvents(rotki.data.db)
+    entries = [
+        {
+            'entry_type': 'asset movement event',
+            'timestamp': 1569924575000,
+            'balance': {'amount': '0.44', 'usd_value': 0},
+            'event_type': 'DEPOSIT',
+            'location': 'BITFINEX',
+            'unique_id': 'BITFINEX-543',
+            'asset': 'ETH',
+        }, {
+            'entry_type': 'asset movement event',
+            'timestamp': 1669924575000,
+            'balance': {'amount': '0.0569', 'usd_value': 0},
+            'event_type': 'WITHDRAWAL',
+            'fee': '0.000004',
+            'location': 'BITFINEX',
+            'unique_id': 'BITFINEX-344',
+            'asset': 'ETH',
+            'fee_asset': 'ETH',
+        },
+    ]
+    for entry in entries:
+        response = requests.put(
+            api_url_for(rotkehlchen_api_server, 'historyeventresource'),
+            json=entry,
+        )
+        result = assert_proper_sync_response_with_result(response)
+        assert 'identifier' in result
+        entry['identifier'] = result['identifier']
+
+    with rotki.data.db.conn.read_ctx() as cursor:
+        saved_events = db.get_history_events(
+            cursor=cursor,
+            filter_query=HistoryEventFilterQuery.make(),
+            has_premium=True,
+            group_by_event_ids=False,
+        )
+    assert len(saved_events) == 3  # including the fee event.
+
```

Comment:
shouldn't you check that the events are what you expect and the fields are populated as you expect?